packages/fxa-content-server/app/scripts/views/index.js

```diff
@@ -45,6 +45,8 @@ class IndexView extends FormView {
       unsafeThirdPartyAuthHTML: this.renderTemplate(ThirdPartyAuth, {
         showSeparator: true,
       }),
+      showSubscriptionTerms:
+        this.isMonitorClient || this.relier.isOAuthNativeRelay(),
```

Comment:
`this.isMonitorClient` looked funky to me when scrolling through just now and I tested this against 123done by adding the client ID to the list and I'm not seeing it render properly for Monitor so, looking into it now.